conduit/data/datasets/audio/ecoacoustics.py

```diff
@@ -113,9 +111,11 @@ def __init__(
         self.specgram_segment_len = specgram_segment_len
         self.resample_rate = resample_rate
         self._n_sgram_segments = int(self._AUDIO_LEN / specgram_segment_len)
-        self.hop_length = hop_length
-        self.num_freq_bins = num_freq_bins
-        self.preprocess_transform = T.Spectrogram(n_fft=num_freq_bins, hop_length=hop_length)
+        if preprocessing_transform is None:
+            preprocessing_transform = T.Spectrogram(n_fft=120, hop_length=60)
+        self.transform = (
```

Comment:
```suggestion
        self.transform = (
```
Rename this to `preprocessing_transform`